packages/components/src/unit-control/stories/index.story.tsx

```diff
@@ -16,7 +16,8 @@ import { CSS_UNITS } from '../utils';
 
 const meta: Meta< typeof UnitControl > = {
 	component: UnitControl,
-	title: 'Components (Experimental)/UnitControl',
+	title: 'Components/Selection & Input/Common/UnitControl',
```

Comment:
Changed from `experimental` to `stable` as per the sitemap. If not this can be reverted to use `experimental` only.